.github/workflows/ci.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install -y clang-format-9
+          sudo pip3 install cmake-format
```

Comment:
You might want to consider pinning this to a specific version, just to make sure things don't change underneath you in weird ways. It is a bit of pain to keep track of if dependabot can't detect this though :/
```
sudo pip3 install cmake-format==0.6.10
```

Reply:
> You might want to consider pinning this to a specific version, just to make sure things don't change underneath you in weird ways. It is a bit of pain to keep track of if dependabot can't detect this though :/

Yeah, good call. I'm usually divided between "I want this to auto-update so that the CI fails when best practices have changed" and "I never want the CI to fail for external reasons", but I think pinning is best here.